download/download.go

```diff
@@ -17,23 +26,155 @@ Download: Downloads files from the Sensitive Data Archive (SDA). If a directory
           download the urls_list.txt file, and then download all files in this
           list. If file urls are given, they will be downloaded as-is.
```

Comment:
Last statement is somewhat confusing from a user perspective. Maybe give more detail for the other cases covered in the code instead?